test/e2e/framework/test_context.go

```diff
@@ -11,8 +11,10 @@ import (
 type contextArray []string
 
 type TestContextType struct {
+	KubeConfigs         []string // KubeConfigs provides an alternative to KubeConfig + KubeContexts
```

Comment:
This needs to be added to `registerFlags` below.  Also I suggest we name the parameter "kubeconfig-path" so usage would be:

  _-kubeconfig-path path1  kubeconfig-path path2_

rather than:

  _-kubeconfigs path1  kubeconfigs path2_

Reply:
I didn't want to make this available from the command line. Just to expose it on the framework for subctl usage.

I could add support for that if we think it'd be beneficial. @skitt said the community and kind is moving away from that model, but I needed support for now on subctl.

I'd skip this, and add it if we think we need it.